feat: blog article why i started documenso (#914)
documenso/documenso: apps/marketing/content/blog/why-i-started-documenso.mdx

---
title: Why I started Documenso
description: TLDR; I started Documenso because I wanted to build a modern tech company in a growing space with a mission bigger than money, I overpaid for a SSL cert 13 years ago, like encryption and want to help make the internet/ world more open.
authorName: 'Timur Ercan'
authorImage: '/blog/blog-author-timur.jpeg'
authorRole: 'Co-Founder'
date: 2024-02-06
Tags:
  - Founders
  - Mission
  - Open Source
---

<figure>
  <MdxNextImage
    src="/blog/burgers.jpeg"
    width="650"
    height="100"
    alt="Burgers, drinks on a table between friends."
  />

  <figcaption className="text-center">

>     No the burger from the story. But it could be as well, the place is pretty generic.

Reviewer: ```suggestion
    Not the burger from the story. But it could be as well, the place is pretty generic.
```

  </figcaption>
</figure>

> TLDR; I started Documenso because I wanted to build a modern tech company in a growing space with a mission bigger than money, I overpaid for a SSL cert 13 years ago, like encryption and want to help make the world/ Internet more open

> It’s hard to pinpoint when I decided to start Documenso. I first uttered the word “Documenso” while sitting in a restaurant with [Felix](https://twitter.com/flxmgdnz), eating a burger and discussing what’s next in late 2022. Shortly after i sat down with a can of caffeine and started building Documenso 0.9. Starting Documenso is the most deliberate business decision I ever made. It was deliberate from the personal side and deliberate from the business side.

Reviewer: ```suggestion
It’s hard to pinpoint when I decided to start Documenso. I first uttered the word “Documenso” while sitting in a restaurant with [Felix](https://twitter.com/flxmgdnz), eating a burger and discussing what’s next in late 2022. Shortly after I sat down with a cup of caffeine and started building Documenso 0.9. Starting Documenso is the most deliberate business decision I ever made. It was deliberate from the personal side and deliberate from the business side.
```

Author: Nah it was actually a can :)

- The global signing market is huge and rapidly growing
- The signing space is huge dominated by one outdated player, to put it bluntly. Outdated in terms of tech, pricing and ecosystem
- The signing space is also ridiculously opaque for a space that is based on open web tech, open encryption tech and open signing standards. Even by closed source standards
- We are currently seeing a renaissance for commercial open source startups, combining venture founder financial with open source mechanics
- Rebuilding a fundamental infrastructure as open source with a meaningful scale, has a profoundly transformative effect for a space
- Working in open source requires you to be open, cooperative and inclusive. It also requires quite a bit of context jumping, “going with the flow” and empathy
- Apart from fixing the signing space, making Documenso successful, would be another domino tile toward open source eating the world, which is great for everyone

> Building a company is so complex, it can’t be planned out. Basing it on great fundamentals and the expected dynmamics it the best founders can do in my humble opinion. After these fundamental decisions you are (almost) just along for the ride and need to focus on solving the “convential” problems of starting a company the best you can. With digital signatures hitting so many point of my personal and professional checklist, this already was a great fit. What got me exited at first though, apart from the perspective of drinking caffeine and coding, was this:

Reviewer: ```suggestion
Building a company is so complex, it can’t be planned out. Basing it on great fundamentals and the expected dynamics it is the best founders can do in my humble opinion. After these fundamental decisions, you are (almost) just along for the ride and need to focus on solving the “conventional” problems of starting a company the best you can. With digital signatures hitting so many points of my personal and professional checklist, this already was a great fit. What got me excited at first though, apart from the perspective of drinking caffeine and coding, was this:
```

> Roughly 13 years ago, I was launching my first product. We obviously wanted SSL encryption on the product site, so I had to buy an SSL certificate. ~$200ish, 2 years validity, from VeriSign I think. Apart from it being ridiculously complicated to get, even back then it bothered me, that we had basically paid for $200 for what is essentially a long number, someone generated. SSL wasn’t even that widespread back then, because it was mainly considered important for ecommerce, no wonder considering it costed so much. “Why would I encrypt a blog?”. Fast forward to today, and everyone can get a free SSL cert courtesy of Let’s Encrypt and browsers basically block unencrypted sites. Mostly even build into hosting plattforms so you barely even notice as a developer.

Reviewer: ```suggestion
Roughly 13 years ago, I was launching my first product. We obviously wanted SSL encryption on the product site, so I had to buy an SSL certificate. ~$200ish, 2 years validity, from VeriSign I think. Apart from it being ridiculously complicated to get, even back then it bothered me, that we had basically paid for $200 for what is essentially a long number, someone generated. SSL wasn’t even that widespread back then, because it was mainly considered important for e-commerce, no wonder considering it cost so much. “Why would I encrypt a blog?”. Fast forward to today, and everyone can get a free SSL cert courtesy of Let’s Encrypt and browsers block unencrypted sites. Mostly even build into hosting platforms so you barely even notice as a developer.
```

> I had forgotten all about that story until I realized, this is where signing is today. A global need, fullfilled only by closed ecosystem, not really state-of-the-art companies, leading to, let’s call it steep prices. I had for so long considered Let’s Encrypt a pillar of the open internet, that I forgot that they weren’t always there. One day someone said, let’s make the internet better. Signing is another domain, that should have had an open ecosystem for a long time. Another parallel to that story is the fact that the cryptographic certificates you need for document signing are also stuck in the “pre Let’s Encrypt world”. Free document signing certificates via "Let’s Sign" are now another todo on the [longterm roadmap](https://documen.so/roadmap) list for open signing ecossytem. Actually effecting this change in any way, is a huge driver for me, personally.

Reviewer: ```suggestion
I had forgotten all about that story until I realized, this is where signing is today. A global need, fulfilled only by closed ecosystem, not really state-of-the-art companies, leading to, let’s call it steep prices. I had for so long considered Let’s Encrypt a pillar of the open internet, that I forgot that they weren’t always there. One day someone said, let’s make the internet better. Signing is another domain, that should have had an open ecosystem for a long time. Another parallel to that story is the fact that the cryptographic certificates you need for document signing are also stuck in the “pre Let’s Encrypt world”. Free document signing certificates via "Let’s Sign" are now another to-do on the [longterm roadmap](https://documen.so/roadmap) list for the open signing ecosystem. Effecting this change in any way is a huge driver for me.
```

> Apart from my personal gripes with the coporate certificate industry, I always found encryption fascinating. It’s such a fundamental force in society when you think about it: Secure Communication, Secure Commerce and even internet native money (Bitcoin) was created using a bit of smart math. All these examples are expressions of very fundamental human behaviours, that should be enabled and protected by open infrastructures.

Reviewer: ```suggestion
Apart from my gripes on the corporate certificate industry, I always found encryption fascinating. It’s such a fundamental force in society when you think about it: Secure Communication, Secure Commerce, and even internet native money (Bitcoin) was created using a bit of smart math. All these examples are expressions of very fundamental human behaviors, that should be enabled and protected by open infrastructures.
```

I never told anyone before, but since starting Documenso I realized that I underestimated the impact and importance of open source for quite some time. When I was in University, I distantly remember my mindset of “yeah open source is nice, but the great, commercially successful products used in the real world are build by closed companies (aka Microsoft)” _shudder_ It was never really a conscious thought, but enough that I started learning MS Silverlight before plain Javascript. It was slowly over time, that I realized that open web standards are superior to closed ones and even later that I understood the same holds true for all software. Open sources fixes something in the economy, I find hard to articulate. I did my best in [commodifying signing].

> To wrap this up, Documenso happens to be the perfect storm of market opportunity, my personal interests and passions. Creating a company people actually want to work for longterm while tackleing these issues is critical side quest of Documenso. This is not only about building the next generation signing tech, it’s also about doing our part to normalize open, healthy, efficient working cultures, tackling relevant problems.

Reviewer: ```suggestion
To wrap this up, Documenso happens to be the perfect storm of market opportunity, my interests, and my passions. Creating a company people want to work for the long term while tackling these issues is a critical side quest of Documenso. This is not only about building the next generation of signing tech, it’s also about doing our part to normalize open, healthy, efficient working cultures, and tackling relevant problems.
```
